andela/ah-frontend-valkyrie
# #162633464 Increase create article coverage

**What does this PR do?**

This PR increases the test coverage for create article

**Description of Task to be completed?**

-  Resolved conflicts
- Add more tests
- Remove snapshots

**How should this be manually tested?**

- Run `npm install` & `npm start` to run the application.
- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**Any background context you want to provide?**

- For redux to work with react, remember to install react-redux by running `npm install react-redux`.

**What are the relevant pivotal tracker stories?**

[#162633452](https://www.pivotaltracker.com/n/projects/2231917/stories/162633464)
